src/app/[locale]/(logged-out)/privacy-policy/components/PrivacyPolicy/PrivaryPolicy.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
```

Comment:
Could you correct the typo in this file's name please `Privary -> Privacy`

Reply:
Thanks